scripts/ci/pipeline-netcore-runtime.yml

```diff
@@ -18,37 +18,73 @@ jobs:
         poolname: Hosted Ubuntu 1604
         manifest: linux-x64
         sed: sed
+      Windows x64:
+        poolname: Hosted VS2017
+        manifest: win-x64
+        sed: sed
       OSX x64:
         poolname: Hosted macOS
         manifest: osx-x64
         sed: gsed
   pool:
     name: $(poolname)
   steps:
-  - script: |  
+  - bash: |  
       brew install autoconf automake libtool gnu-sed
     displayName: 'Prepare OSX dependencies'
     condition: eq(variables['poolname'], 'Hosted macOS')
 
-  - script: |  
+  - bash: |  
       sudo apt update
       sudo apt -y install nuget build-essential libtool libtool-bin cmake gettext dotnet-sdk-2.2
     displayName: 'Prepare Linux dependencies'
     condition: eq(variables['poolname'], 'Hosted Ubuntu 1604')
   
+  - script: |
+      choco install msys2 --params="/InstallDir:%CD:~0,2%\msys64 /NoUpdate /NoPath"
+      %CD:~0,2%\msys64\usr\bin\pacman --noconfirm -Syyuu
```

Comment:
How about just `\msys64` without the `%CD:~0,2%`?
Does something really require a full path there?